gravitee-apim-rest-api/gravitee-apim-rest-api-management-v2/gravitee-apim-rest-api-management-v2-rest/src/main/resources/openapi/management-openapi-v2.yaml

```diff
@@ -6100,6 +6142,11 @@ components:
                                 type: array
                                 items:
                                     $ref: "#/components/schemas/Page"
+                            breadcrumb:
```

Comment:
```suggestion
                            breadcrumbs:
```
WDYT ?

Reply:
The ensemble of elements create a breadcrumb, so I would keep it singular.